README.md

```diff
@@ -1 +1,23 @@
-# platatus
+# Project Platatus
+
+*tomayto, tomahto*, or also **platform-status**
+
+## Goal
+
+Platatus gives developers a cross-platform and cross-browser roadmap for browser features, from standardization, development to release. Not only being a channel to raise awareness, it can also drive adoption and improve the vital feedback loop Firefox's platform needs to have with web developers. For most developers it will be a reference to discover and learn, but it has the potential to provide the entry points for radical participation on Mozilla's platform development.
+
+## Why
+
+Existing web compatibilty references, like [caniuse](http://caniuse.com/) focus on cross-browser compatibilty but don't cover un-released features and lag behind on features that already shipped. Major browser vendors ([Edge](https://dev.modern.ie/platform/status/), [Chrome](https://www.chromestatus.com/features), [Webkit](https://www.webkit.org/status.html)) are publicising their own platform status with references to their bug tracking and bug owners. Similar to Platatus' goal, Chrome tracks consensus/enthusiasm from other browsers using sources that range from bugs, to IRC logs to tweets.
+
+Platatus can send an open and clear message to developers on the state of Mozilla's platform, starting from conception of a new features to release.
+
+## What is success
+
+- Developers discover new platform features early in the process, try them out and provide feedback
+- Developers adopt bleeding edge standards because they get clear signals from browser vendors
```

Comment:
From what's written above, it sounds like developers are already getting clear signals about implementation status from other browser vendors. Is the "clear signals from browser vendors" meant about Mozilla specifically? Or maybe it means that there will be clear signals of intent prior to actual implementation?